oar/core/util.py

```diff
@@ -0,0 +1,22 @@
+def is_valid_z_release(version):
+    strs = version.split(".")
+    if len(strs) != 3:
+        return False
+
+    valid = True
+    for v in strs:
+        if not v.isdigit():
+            valid = False
+            break
+
+    return valid
+
+
+def get_y_release(version):
```

Comment:
@rioliu-rh for this method, raise Exception or return None?
I ask it because
1) I do not find the catch exception of method which call this method.
2) for util, we could return something, not raise exception. for other function in core, raise exception.

Reply:
@kuiwang02 argree, input validation should not be handled at util function, it should be done at invoker level

Reply:
done